### PR TITLE
rename poke shop to poke mart

### LINKED ISSUE #26: rename `poké shop` to `poké mart`
https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9_Mart

### PR DESCRIPTION
**Pull request by vcapretz**
closes #26

### CODE REVIEW

File: bin/pokemon-shop.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ module.exports = async () => {
         },
         new inquirer.Separator(),
         {
-          name: chalk.gray('Exit Poké shop'),
+          name: chalk.gray('Exit Poké mart'),
```
Comment (bukinoshita) [inline bin/pokemon-shop.js]: Can you change `mart` to `Mart`?

**Comment by vcapretz** [comment]
@bukinoshita should we rename the pokemon-shop project as well?

**Comment by bukinoshita** [comment]
you mean the file? if so, yes.

**Comment by vcapretz** [comment]
done! everything renamed to pokemon mart :smile:

**Comment by bukinoshita** [comment]
awesome @vcapretz, ty!

**Pull request merged**